docs/apis/plugintypes/theme/_examples/lang.md

```diff
@@ -0,0 +1,15 @@
+<!-- markdownlint-disable first-line-heading -->
+<!-- cspell:ignore choosereadme, configtitle -->
+
+Each plugin must define a set of language strings with, at a minimum, an English translation. These are specified in the plugin's lang/en directory in a file named after the plugin.
+
+Language strings for the plugin. Required strings:
+
+- **pluginname** - name of plugin.
+- **choosereadme** - descriptive text displayed beneath the theme information dialog screenshot.
+- **configtitle** - settings text for this type of plugin.
```

Comment:
Backticks around the strings (bold optional):

```suggestion
- **`pluginname`** - name of plugin.
- **`choosereadme`** - descriptive text displayed beneath the theme information dialog screenshot.
- **`configtitle`** - settings text for this type of plugin.
```

This negates the need to put them in cspell ignore too.